packages/snaps-sdk/src/types/permissions.ts

```diff
@@ -0,0 +1,56 @@
+import type { JsonRpcRequest } from '@metamask/utils';
+
+import type { ChainId } from './caip';
+
+export type EmptyObject = Record<string, never>;
+
+export type Cronjob = {
+  expression: string;
```

Comment:
I believe there is another way to specify these, but it might be broken..

Reply:
Yeah, I don't think this has ever worked. This matches the type as inferred by Superstruct.